client/app/lib/push-utility.js

```diff
@@ -15,12 +16,10 @@ export default async function registerForPushNotificationsAsync() {
   console.log('Final Status: '+finalStatus);
   // Stop here if the user did not grant permissions
   if (finalStatus !== 'granted') {
-    return;
+    return undefined;
```

Comment:
So is the plan here to change this up later so the token can be retrieved whether or not the user grants push notification permission? Because it's going to be needed to get the allergen and dietary restriction filters as well

Reply:
Well I suppose if the token is undefined we can kill the app until they grant permissions? :joy:

Reply:
well yeah but they shouldn't need to give permission for push notifications in order to set their allergens and filters right?

Reply:
Yeah it should probs be changed later along with a change to check whether the user has allowed push notifications, instead of relying on the token being undefined to tell us that the user doesn't allow them (which is what I think it's doing now?)